<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="5" ignored="0" retried="3" total="26" passed="17">
  <reporter-output>
    <line>
      <![CDATA[Clicking on Login Link By.linkText: Log in]]>
    </line>
    <line>
      <![CDATA[Enter email prime123@gmail.com to email field By.id: Email]]>
    </line>
    <line>
      <![CDATA[Enter password prime123 to password field By.name: Password]]>
    </line>
    <line>
      <![CDATA[Clicking on Login Button By.xpath: //button[contains(text(),'Log in')]]]>
    </line>
    <line>
      <![CDATA[Clicking on Login Link By.linkText: Log in]]>
    </line>
    <line>
      <![CDATA[Getting Welcome Text ]]>
    </line>
  </reporter-output>
  <suite name="Command line suite" duration-ms="8542" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:25 BST">
    <groups>
      <group name="regression">
        <method signature="TestNGGrouping.loginTest1()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="loginTest1" class="com.nopcommerce.demo.testng.TestNGGrouping"/>
        <method signature="TestNGGrouping.loginTest2()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="loginTest2" class="com.nopcommerce.demo.testng.TestNGGrouping"/>
        <method signature="TestNGGrouping.loginTest3()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="loginTest3" class="com.nopcommerce.demo.testng.TestNGGrouping"/>
        <method signature="TestNGGrouping.loginTest4()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="loginTest4" class="com.nopcommerce.demo.testng.TestNGGrouping"/>
        <method signature="TestNGGrouping.regressionTest1()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="regressionTest1" class="com.nopcommerce.demo.testng.TestNGGrouping"/>
        <method signature="TestNGGrouping.regressionTest2()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="regressionTest2" class="com.nopcommerce.demo.testng.TestNGGrouping"/>
        <method signature="TestNGGrouping.regressionTest3()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="regressionTest3" class="com.nopcommerce.demo.testng.TestNGGrouping"/>
      </group> <!-- regression -->
      <group name="regression1">
        <method signature="TestNGGrouping.loginTest3()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="loginTest3" class="com.nopcommerce.demo.testng.TestNGGrouping"/>
        <method signature="TestNGGrouping.loginTest4()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="loginTest4" class="com.nopcommerce.demo.testng.TestNGGrouping"/>
        <method signature="TestNGGrouping.regressionTest1()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="regressionTest1" class="com.nopcommerce.demo.testng.TestNGGrouping"/>
        <method signature="TestNGGrouping.regressionTest2()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="regressionTest2" class="com.nopcommerce.demo.testng.TestNGGrouping"/>
        <method signature="TestNGGrouping.regressionTest3()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="regressionTest3" class="com.nopcommerce.demo.testng.TestNGGrouping"/>
      </group> <!-- regression1 -->
      <group name="smoke">
        <method signature="TestNGGrouping.loginTest2()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="loginTest2" class="com.nopcommerce.demo.testng.TestNGGrouping"/>
        <method signature="TestNGGrouping.loginTest3()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="loginTest3" class="com.nopcommerce.demo.testng.TestNGGrouping"/>
        <method signature="TestNGGrouping.loginTest4()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="loginTest4" class="com.nopcommerce.demo.testng.TestNGGrouping"/>
      </group> <!-- smoke -->
      <group name="sanity">
        <method signature="TestNGGrouping.loginTest1()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="loginTest1" class="com.nopcommerce.demo.testng.TestNGGrouping"/>
        <method signature="TestNGGrouping.loginTest2()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="loginTest2" class="com.nopcommerce.demo.testng.TestNGGrouping"/>
      </group> <!-- sanity -->
    </groups>
    <test name="Command line test" duration-ms="8542" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:25 BST">
      <class name="com.nopcommerce.demo.testng.TestNGPriority">
        <test-method status="PASS" signature="test()[pri:1, instance:com.nopcommerce.demo.testng.TestNGPriority@7b69c6ba]" name="test" duration-ms="0" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method status="PASS" signature="test2()[pri:1, instance:com.nopcommerce.demo.testng.TestNGPriority@7b69c6ba]" name="test2" duration-ms="0" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
        <test-method status="PASS" signature="test1()[pri:3, instance:com.nopcommerce.demo.testng.TestNGPriority@7b69c6ba]" name="test1" duration-ms="0" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
      </class> <!-- com.nopcommerce.demo.testng.TestNGPriority -->
      <class name="com.nopcommerce.demo.testng.TestNGRetryFailedDemo">
        <test-method status="PASS" signature="test1()[pri:0, instance:com.nopcommerce.demo.testng.TestNGRetryFailedDemo@46daef40]" name="test1" duration-ms="0" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method status="PASS" signature="test2()[pri:0, instance:com.nopcommerce.demo.testng.TestNGRetryFailedDemo@46daef40]" name="test2" duration-ms="1" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
        <test-method status="SKIP" signature="test3()[pri:0, instance:com.nopcommerce.demo.testng.TestNGRetryFailedDemo@46daef40]" name="test3" duration-ms="1" retried="true" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at com.nopcommerce.demo.testng.TestNGRetryFailedDemo.test3(TestNGRetryFailedDemo.java:24)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test3 -->
        <test-method status="SKIP" signature="test3()[pri:0, instance:com.nopcommerce.demo.testng.TestNGRetryFailedDemo@46daef40]" name="test3" duration-ms="0" retried="true" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at com.nopcommerce.demo.testng.TestNGRetryFailedDemo.test3(TestNGRetryFailedDemo.java:24)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:262)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test3 -->
        <test-method status="SKIP" signature="test3()[pri:0, instance:com.nopcommerce.demo.testng.TestNGRetryFailedDemo@46daef40]" name="test3" duration-ms="0" retried="true" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at com.nopcommerce.demo.testng.TestNGRetryFailedDemo.test3(TestNGRetryFailedDemo.java:24)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:262)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test3 -->
        <test-method status="FAIL" signature="test3()[pri:0, instance:com.nopcommerce.demo.testng.TestNGRetryFailedDemo@46daef40]" name="test3" duration-ms="1" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at com.nopcommerce.demo.testng.TestNGRetryFailedDemo.test3(TestNGRetryFailedDemo.java:24)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:262)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test3 -->
      </class> <!-- com.nopcommerce.demo.testng.TestNGRetryFailedDemo -->
      <class name="com.nopcommerce.demo.testng.TestNGSoftAssert">
        <test-method status="FAIL" signature="hardAssertExample()[pri:0, instance:com.nopcommerce.demo.testng.TestNGSoftAssert@12f41634]" name="hardAssertExample" duration-ms="0" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at com.nopcommerce.demo.testng.TestNGSoftAssert.hardAssertExample(TestNGSoftAssert.java:18)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- hardAssertExample -->
        <test-method status="FAIL" signature="softAssertExample()[pri:0, instance:com.nopcommerce.demo.testng.TestNGSoftAssert@12f41634]" name="softAssertExample" duration-ms="2" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
First Assert expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
First Assert expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at com.nopcommerce.demo.testng.TestNGSoftAssert.softAssertExample(TestNGSoftAssert.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- softAssertExample -->
      </class> <!-- com.nopcommerce.demo.testng.TestNGSoftAssert -->
      <class name="com.nopcommerce.demo.testng.TestNGDemo">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:com.nopcommerce.demo.testng.TestNGDemo@5f3a4b84]" name="beforeTest" is-config="true" duration-ms="5" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:com.nopcommerce.demo.testng.TestNGDemo@5f3a4b84]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="test1()[pri:0, instance:com.nopcommerce.demo.testng.TestNGDemo@5f3a4b84]" name="test1" duration-ms="1" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.nopcommerce.demo.testng.TestNGDemo@5f3a4b84]" name="afterMethod" is-config="true" duration-ms="0" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:com.nopcommerce.demo.testng.TestNGDemo@5f3a4b84]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="test2()[pri:0, instance:com.nopcommerce.demo.testng.TestNGDemo@5f3a4b84]" name="test2" duration-ms="1" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.nopcommerce.demo.testng.TestNGDemo@5f3a4b84]" name="afterMethod" is-config="true" duration-ms="0" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:com.nopcommerce.demo.testng.TestNGDemo@5f3a4b84]" name="afterTest" is-config="true" duration-ms="1" started-at="2023-07-18T10:36:25 BST" finished-at="2023-07-18T10:36:25 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- com.nopcommerce.demo.testng.TestNGDemo -->
      <class name="com.nopcommerce.demo.testsuite.LoginPageTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.nopcommerce.demo.testsuite.LoginPageTest@8646db9]" name="setUp" is-config="true" duration-ms="3275" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:20 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="verifyErrorMessageWithInvalidCredentials()[pri:0, instance:com.nopcommerce.demo.testsuite.LoginPageTest@8646db9]" name="verifyErrorMessageWithInvalidCredentials" duration-ms="1566" started-at="2023-07-18T10:36:20 BST" finished-at="2023-07-18T10:36:22 BST">
          <reporter-output>
            <line>
              <![CDATA[Clicking on Login Link By.linkText: Log in]]>
            </line>
            <line>
              <![CDATA[Enter email prime123@gmail.com to email field By.id: Email]]>
            </line>
            <line>
              <![CDATA[Enter password prime123 to password field By.name: Password]]>
            </line>
            <line>
              <![CDATA[Clicking on Login Button By.xpath: //button[contains(text(),'Log in')]]]>
            </line>
          </reporter-output>
        </test-method> <!-- verifyErrorMessageWithInvalidCredentials -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.nopcommerce.demo.testsuite.LoginPageTest@8646db9]" name="tearDown" is-config="true" duration-ms="597" started-at="2023-07-18T10:36:22 BST" finished-at="2023-07-18T10:36:22 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.nopcommerce.demo.testsuite.LoginPageTest@8646db9]" name="setUp" is-config="true" duration-ms="1713" started-at="2023-07-18T10:36:22 BST" finished-at="2023-07-18T10:36:24 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="verifyUserShouldNavigateToLoginPageSuccessfully()[pri:0, instance:com.nopcommerce.demo.testsuite.LoginPageTest@8646db9]" name="verifyUserShouldNavigateToLoginPageSuccessfully" duration-ms="713" started-at="2023-07-18T10:36:24 BST" finished-at="2023-07-18T10:36:25 BST">
          <reporter-output>
            <line>
              <![CDATA[Clicking on Login Link By.linkText: Log in]]>
            </line>
            <line>
              <![CDATA[Getting Welcome Text ]]>
            </line>
          </reporter-output>
        </test-method> <!-- verifyUserShouldNavigateToLoginPageSuccessfully -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.nopcommerce.demo.testsuite.LoginPageTest@8646db9]" name="tearDown" is-config="true" duration-ms="622" started-at="2023-07-18T10:36:25 BST" finished-at="2023-07-18T10:36:25 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.nopcommerce.demo.testsuite.LoginPageTest -->
      <class name="com.nopcommerce.demo.testng.TestNGParameterDemo">
        <test-method status="FAIL" signature="test(java.lang.String,java.lang.String)[pri:0, instance:com.nopcommerce.demo.testng.TestNGParameterDemo@44c8afef]" name="test" duration-ms="0" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Parameter 'surname' is required by @Test on method test but has not been marked @Optional or defined
]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Parameter 'surname' is required by @Test on method test but has not been marked @Optional or defined

at org.testng.internal.Parameters.createParams(Parameters.java:267)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:355)
at org.testng.internal.Parameters.createParameters(Parameters.java:660)
at org.testng.internal.Parameters.handleParameters(Parameters.java:856)
at org.testng.internal.Parameters.handleParameters(Parameters.java:704)
at org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:70)
at org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:49)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:904)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
      </class> <!-- com.nopcommerce.demo.testng.TestNGParameterDemo -->
      <class name="com.nopcommerce.demo.testng.TestNGGrouping">
        <test-method status="PASS" signature="loginTest1()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="loginTest1" duration-ms="0" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest1 -->
        <test-method status="PASS" signature="loginTest2()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="loginTest2" duration-ms="1" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest2 -->
        <test-method status="PASS" signature="loginTest4()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="loginTest4" duration-ms="0" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest4 -->
        <test-method status="PASS" signature="regressionTest1()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="regressionTest1" duration-ms="1" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- regressionTest1 -->
        <test-method status="PASS" signature="loginTest3()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="loginTest3" duration-ms="0" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest3 -->
        <test-method status="PASS" signature="regressionTest3()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="regressionTest3" duration-ms="0" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- regressionTest3 -->
        <test-method status="PASS" signature="regressionTest2()[pri:0, instance:com.nopcommerce.demo.testng.TestNGGrouping@27f723]" name="regressionTest2" duration-ms="0" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- regressionTest2 -->
      </class> <!-- com.nopcommerce.demo.testng.TestNGGrouping -->
      <class name="com.nopcommerce.demo.testng.TestNGListenersDemo">
        <test-method status="FAIL" signature="test2()[pri:0, instance:com.nopcommerce.demo.testng.TestNGListenersDemo@4923ab24]" name="test2" duration-ms="2" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.fail(Assert.java:115)
at com.nopcommerce.demo.testng.TestNGListenersDemo.test2(TestNGListenersDemo.java:22)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
        <test-method status="PASS" signature="test1()[pri:0, instance:com.nopcommerce.demo.testng.TestNGListenersDemo@4923ab24]" name="test1" duration-ms="0" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method status="SKIP" signature="test3()[pri:0, instance:com.nopcommerce.demo.testng.TestNGListenersDemo@4923ab24]" name="test3" duration-ms="0" started-at="2023-07-18T10:36:17 BST" finished-at="2023-07-18T10:36:17 BST">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Test Skipped]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Test Skipped
at com.nopcommerce.demo.testng.TestNGListenersDemo.test3(TestNGListenersDemo.java:28)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test3 -->
      </class> <!-- com.nopcommerce.demo.testng.TestNGListenersDemo -->
    </test> <!-- Command line test -->
  </suite> <!-- Command line suite -->
</testng-results>
